test/annotations/group.test.js

```diff
@@ -10,4 +10,12 @@ describe('#group', function () {
     assert.deepEqual(group.parse('group'), ['group'])
     assert.deepEqual(group.parse('GRoup'), ['group'])
   })
+
+  it('should parse a description from subsequent lines', function () {
+    var item = {}
+    assert.deepEqual(group.parse('group\ndescription', item), ['group'])
+    assert.deepEqual(item, {'groupDescriptions': {
+      'group': 'description'
+    }})
+  })
```

Comment:
This will need more testing. 

You went with the white-space version finally. 

What happens when used as follow:
```
/// @group name description
```

```
////
/// This is a poster comment.
/// @group name
/// A group description
/// @author Hugo Giraudel
////
```